### ISSUE TITLE
Pattern Detection: SMA / EMA as resistance or support

### ISSUE THREAD

**Issue by Haehnchen** [Haehnchen/crypto-trading-bot#2]
SMA and / or EMA are common for resistance and support.

Action
- Enter on dip
- Enter on break a "suitable price line"
- Check candle wicks
- Valid range between SMA and EMA

![pattern_ema_support](https://user-images.githubusercontent.com/1011712/46167662-b0afe300-c296-11e8-8790-94d5cf7a8496.png)

**Comment by miemczyk** [comment]
Good idea. However, couple of basic strategies in the sample strategies folder such as a basic SMA cross and a short first candle to open under the SMA 7 would be idea for people to begin building on to achieve the above? Also, have you any issue with someone setting up a discord in order to offer a community space for users to come together to help out on this development?
